sdk/cosmos/azure-cosmos-benchmark/src/main/resources/encryption_setting.properties

```diff
@@ -0,0 +1,5 @@
+#### Azure Key Vault config
+#TenantId = REPLACEME
+#ClientId = REPLACEME
+#ClientSecret = REPLACEME
+#KeyVaultMasterKeyUrl = https://REPLACEME.vault.azure.net/keys/REPLACEME/REPLACEME
```

Comment:
benchmark tool at the end runs as a standalone jar, how does this template file in the resource folder helps?

shouldn't this be outside of the source code, maybe in a small docs folder?

Reply:
Changed the method which is consuming this to non static , now this file will be accessed directly by the code via AsyncEncryptionBenchmark.class.getClassLoader().getResourceAsStream("encryption_setting.properties");) without having this file manually need to put on the machine where the jar is

Reply:
if the jar is already created, then this file in the template format is in the jar without the proper uri, etc (you have replaceme) which means the end user has to modify the file in the jar.

the configuration has to be independent of the binary. that's the reason I suggested you should have this file outside of the source code.

@simplynaveen20  any specific reason you want this file to be in the source code and not in some docs or config folder outside of the source code?

Reply:
This the code resource so it should go under resource. User can change the file and create the single jar ,then he need not to worry about putting it explicitly in the same local location as jar, this will avoid overhead.

Reply:
ok, thanks Naveen for the explanation.